token/cli/src/main.rs

```diff
@@ -972,6 +972,12 @@ fn main() {
     let mut wallet_manager = None;
     let mut bulk_signers: Vec<Option<Box<dyn Signer>>> = Vec::new();
 
+    let (sub_command, sub_matches) = app_matches.subcommand();
+    let matches = if let Some(matches) = sub_matches.as_ref() {
+        matches
+    } else {
+        &app_matches
+    };
```

Comment:
I don't think there are any situations where we get here with `sub_matches.is_none()`, are there?
As such, if we want to avoid the unwrap for safety first, I think it might be more clear to do something like:
```
let matches = sub_matches.unwrap_or_else(|| clap::Error{ ... }.exit());
```

Reply:
Ah, nice catch! I was thinking if the command line contained no sub-command, but one of the global args that clap doesn't handle, it'd sneak by.  That doesn't seem to be the case, though.

Reply:
I just went with an `unwrap()` since I couldn't find a way to hit it.